src/test/java/seedu/address/model/person/UniquePersonListTest.java

```diff
@@ -167,4 +169,25 @@ public void asUnmodifiableObservableList_modifyList_throwsUnsupportedOperationEx
         assertThrows(UnsupportedOperationException.class, ()
             -> uniquePersonList.asUnmodifiableObservableList().remove(0));
     }
+
+    @Test
+    public void iterator_success() {
+        uniquePersonList.add(ALICE);
+        Iterator<Person> iterator = uniquePersonList.iterator();
+        assertTrue(iterator.hasNext());
+        assertEquals(iterator.next(), ALICE);
+        assertFalse(iterator.hasNext());
+    }
+
+    @Test
+    public void hashCode_success() {
```

Comment:
check for equals too! this is more impt than not equals as it's part of the hashcode  contract.